tests/add_subdirectory/vk_dispatch_table.c

```diff
@@ -3,21 +3,23 @@
 // Copyright 2023 LunarG, Inc.
 //
 // SPDX-License-Identifier: Apache-2.0
-#include <vulkan/utility/vul_dispatch_table.h>
+#include <vulkan/utility/vk_dispatch_table.h>
 
-PFN_vkVoidFunction local_vkGetInstanceProcAddr(VkInstance instance, const char *pName) {
+// NOLINTBEGIN
+
+PFN_vkVoidFunction VKAPI_PTR local_vkGetInstanceProcAddr(VkInstance instance, const char *pName) {
     (void)instance;
     (void)pName;
     return NULL;
 }
 
-PFN_vkVoidFunction local_vkGetDeviceProcAddr(VkDevice device, const char *pName) {
+PFN_vkVoidFunction VKAPI_PTR local_vkGetDeviceProcAddr(VkDevice device, const char *pName) {
```

Comment:
FYI @charles-lunarg I had to add `VKAPI_PTR` otherwise my build failed to compile for x86

Reply:
That sounds right, any function pointer getting passed around on windows need VKAPI_PTR to get the correct calling convention.